packages/modules/order/src/utils/actions/reinstate-item.ts

```diff
@@ -0,0 +1,64 @@
+import { MathBN, MedusaError, isDefined } from "@medusajs/utils"
+import { ChangeActionType } from "../action-key"
+import { OrderChangeProcessing } from "../calculate-order-change"
+import { setActionReference } from "../set-action-reference"
+
+OrderChangeProcessing.registerActionType(ChangeActionType.REINSTATE_ITEM, {
```

Comment:
question: What is this case for?

Reply:
When an item that was claimed was cancelled/unclaimed.